packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -624,6 +625,20 @@ Future<void> _createStubAppFramework(File outputFile, Environment environment,
   _signFramework(environment, outputFile.path, BuildMode.debug);
 }
 
+void _stripDebugSymbols(Environment environment, String binaryPath, BuildMode buildMode) {
```

Comment:
This will strip even if `--no-strip` is passed via `--extra-gen-snapshot-options` https://github.com/flutter/flutter/pull/75243